### PR TITLE
Fixed Travis warnings 

### PR DESCRIPTION
**Pull request by Weirdo914**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -34,3 +33,5 @@ script:
 
 after_success:
   coveralls
+
```
Comment (atb00ker) [inline .travis.yml]: Please do not add lines at the end of the file. 
It's highly encouraged to not make any unnecessary white space changes.

File: .travis.yml [hunk 2]
```diff
@@ -1,9 +1,8 @@
 language: python
-sudo: false
 cache: pip
 
 python:
-  - "3.4"
+  - "3.5"
```
Comment (atb00ker) [inline .travis.yml]: Any particular reason this change was made in this pull request?

Comment (Weirdo914) [reply .travis.yml]: yes it was causing problems in the build

**Comment by Weirdo914** [comment]
Should I make changes and then send?

**Comment by Weirdo914** [comment]
Removed the unnecessary space and squashed the changes

File: .travis.yml (hunk 2 above)
Comment (atb00ker) [reply .travis.yml]: Okay. Thanks!

**Pull request merged**